salesagility/SuiteCRM
# Fixed #5749 - [outgoing emails] From field incorrectly filled

## Description



Issue reference: #5749 

## How To Test This

1. Setup personal inbound/outbound email.
2. Send an email and check that the from field is correctly filled.
3. Repeat step 1/2 for other types of outbound such as group and system.

## Types of changes


### Final checklist
